packages/dds/task-manager/src/interfaces.ts

```diff
@@ -19,11 +19,18 @@ export interface ITaskManagerEvents extends ISharedObjectEvents {
 
 export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
     /**
-     * Try to lock the task.  Promise resolves when the lock is acquired, or rejects if we are removed from the
-     * queue without acquiring the lock for any reason.
+     * Volunteer to lock the task.  Promise resolves true when the lock is acquired. It rejects if we are removed from
+     * the queue without acquiring the lock for any reason, such as disconnecting or abandoning the task while in queue.
```

Comment:
```suggestion
     * Volunteer for the task.  Promise resolves true when the task is assigned to the local client. It rejects if the local client is removed from
     * the queue without being assigned the task for any reason, such as disconnecting or abandoning the task while in queue.
```

Reply:
And similar language for `subscribeToTask` and `assignedTask`, avoiding the term "lock"